packages/api/endpoints/launchpadSaml.js

```diff
@@ -42,6 +42,48 @@ const launchpadPublicCertificate = async (launchpadPublicMetadataPath) => {
   );
 };
 
+/**
+ * Validates the SAML user Group includes the configured authorized User Group.
+ *
+ * @param {string} samlUserGroup -  Saml response string e.g.:
+       'cn=wrongUserGroup,ou=254886,ou=ROLES,ou=Groups,dc=nasa,dc=gov'.
+ * @param {string} authorizedGroup - Cumulus oauth user group.
+ * @returns {boolean} True if samlUserGroup includes the authorizedUserGroup.
+ */
+const authorizedUserGroup = (samlUserGroup, authorizedGroup) => {
+  const matcher = new RegExp(`cn=${authorizedGroup}`);
+  return matcher.test(samlUserGroup);
```

Comment:
Not totally sure how this test works. What if there is more than one match for the regex? It is possible to be in multiple user groups.

Reply:
It is possible to be in more than one group, but Oh.  What does the return value look like for a user in more than one group? Do you have an example?
Is it  `'cn=FirstUserGroup,secondUserGroup,ou=254886,ou=ROLES,ou=Groups,dc=nasa,dc=gov'`
 or `'cn=FirstUserGroup,cn=secondUserGroup,ou=254886,ou=ROLES,ou=Groups,dc=nasa,dc=gov'`

Reply:
I was assuming the second

Reply:
yeah I believe more like the second. Can we add a test for that so we know the behavior?

Reply:
I need to know what the saml result looks like in the case of a user with multiple groups in order to write the test.  If it's the first, then my logic is bad and I need to update the authorizedUserGroup, if it's the second, I can update the test.

Reply:
Updated in 83626400a